index.html

```diff
@@ -233,6 +233,58 @@ <h2 class="footer-heading">🟠 Skills</h2>
       </div>
     </div>
   </section>
+  <section class="contact">
+      <h2 class="conclusion-txt">
+      I'm always interested in 
+      hearing about new projects, 
+      so if you'd like to chat please 
+      get in touch.
+    </h2>
+    <form class="form" action="https://formspree.io/html/" method="post">
+      <div class="input-username">
+        <input
+      class="input-1"
+      type="text" value="Arnold"
+      maxlength="30"
+      placeholder="Full name"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      <div class="input-username">
+        <input 
+      class="input-1"
+      type="email"
+      placeholder="Email address"
+      required
+      >
+      <hr class="line-z">
+      </div>
+      
+      <textarea class="textarea" maxlength="500">Enter text here</textarea>
```

Comment:
- [ ] Kindly 😃 try to stick to the Figma Design, so please here I see that the textarae is outside of the align with from content, so please try to correct that to align it with the form 😊
- [ ] Kindly try to fix the border of the elements in the form to be like the Figma design 😊as you see the border in yours is bolder than the Figma Design 💯

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/221972428-970dc465-0edb-42c0-b32b-8d7963a8c103.png)   |    ![image](https://user-images.githubusercontent.com/83360636/221972929-b5dbc6a9-d059-4dcb-baab-fc03a3b47ea4.png)   |